### DISCUSSION TITLE
Welcome to Movement Discussions!

### DISCUSSION THREAD

**Discussion by l-monninger** [movementlabsxyz/movement#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamesatomc** [comment]
Can you make an engine movement sdk that can create its own chains? After all, the Cosmos side developed the engine to create chains. And it's a good result from the ecosystem that I observed and tried to play with move ment sdk. The movement sdk has standards for writing move sui and aptos. I think we should do it quickly. I will see many developers interested in building something that uses moveVM and evm because it creates new use cases and is very beneficial to developers who want to build their own chains.
